File: C/Simple-Programming-Problems/001_Elementary/005_Num sum modulo 3 or 5/NumSumModulo.c
#include <stdio.h>

/*
* Modify the previous program such that only multiples of three or five are considered in the sum, e.g. 3, 5, 6, 9, 10, 12, 15 for n=17
*/

int main(int argc, char * argv[])
{
    int numberBuffer=0;
    int isDebug=0;
    
    printf("Enter an integer: ");
    scanf("%d",&numberBuffer);

    int sum=0;
    for(int i=numberBuffer;i>0;i--)
    {
        if((i%3==0)||(i%5==0))
        {
            if(isDebug)
            {
                printf("[DEBUG]: Sum is %d + %d\n",sum,i);
            }
            sum+=i;
        } else {
            continue;
        }
    }
    printf("The sum of each number that is a multiple of 3 or 5 in %d is: %d \n",numberBuffer,sum);

    return 0;
}

File: Python/Simple-Programming-Problems/001_Elementary/005_Num sum modulo 3 or 5/NumSumModulo.py
#!/bin/python3

"""
Task:
Modify the previous program such that only multiples of three or five are considered in the sum, e.g. 3, 5, 6, 9, 10, 12, 15 for n=17
"""

numberBuffer=0
isDebug=False

numberBuffer=int(input("Enter an integer: "))

sum=0
for i in range(numberBuffer):
    if((i%3==0)or(i%5==0)):
        if(isDebug):
            print(f"[DEBUG]: Sum is {sum} + {i}")
        sum+=i
    else:
        continue

print(f"The sum of all numbers that are multiples of 3 or 5 in {numberBuffer} is: {sum} \n")